webapp/channels/src/components/signup/signup.tsx

```diff
@@ -830,7 +765,35 @@ const Signup = ({onCustomizeHeader}: SignupProps) => {
                                         info={passwordInfo as string}
                                         error={passwordError}
                                     />
-                                    {getNewsletterCheck()}
+                                    <CheckInput
+                                        id='signup-body-card-form-check-terms-and-privacy'
+                                        ariaLabel={formatMessage({id: 'signup.terms_and_privacy.checkmark.box', defaultMessage: 'Terms and privacy policy checkbox'})}
+                                        name='newsletter'
+                                        onChange={() => setSubscribeToSecurityNewsletter(!subscribeToSecurityNewsletter)}
```

Comment:
`subscribeToSecurityNewsletter` is triggering a call to the CWS to save the email which was intended to later sync with Snowflake/Marketo. Given we are no longer doing that we need to remove the function that executes that in this componenet here; https://github.com/mattermost/mattermost/blob/402b70e645deb64bce0c5a8a72e61e1c409cdc8a/webapp/channels/src/components/signup/signup.tsx#L599 and all the underlying calls up to Client4 to `/subscribe-newsletter` endpoint to the server and all the server side handling of that api as we won't be needed it anymore. 

Later this can also be deprecated from the CWS but that would be another change.

Reply:
Pushed the changes for the cleanup in the server side. Let me know if something was missed